### ISSUE TITLE
Draft of blog post to go along with Survey

### ISSUE THREAD

**Issue by mhdawson** [nodejs/next-10#52]
### Next 10 years of Node.js - Understanding the the needs of the Node.js constituencies

****TLDR; We need your help to make sure the Next 10 years of Node.js are as successful as the first.**  We are launching a survey, you can take [here]() to help us do that. To get a bit more context on why this survey is important, read on….**

Node.js had a very successful first 10 years of Node.js and the project is working to make the next 10 years even better. As part of that we’ve kicked off the Next-10 effort to document what we think is important for that to happen. You can follow the ongoing work of that team in this repo: https://github.com/nodejs/next-10.  

Without a handy crystal ball, it turns out that it’s a lot harder than just diving in and discussing our favorite technologies to see what the keys to success are going to be. Are things like WebAssembly, Typescript, etc. important to the people who use Node.js ?  I guess we need to better understand/document who uses Node.js first…..

So far the team has spent most of its time laying the foundation on which we hope to base discussions around specific technologies.

We started by documenting our understanding of the project’s technical values as these will help us balance different aspects when necessary: https://github.com/nodejs/node/blob/master/doc/guides/technical-values.md.  It’s not as simple as X overrides Y but instead highlight what key values need to be factored into decisions.  For example, there was consensus that good developer experience has been a key part of the success of Node.js and that it’s important for future success that we maintain that.

Next the team documented the Node.js “Constituencies”. The people/groups who have a stake in the Node.js ecosystem.  We captured these in [CONSTITUENCIES.md](https://github.com/nodejs/next-10/blob/main/CONSTITUENCIES.md) are include:

* Direct end users
* Application operators
* Application Developers
* Back-end server authors
* Library/package authors
* Node.js core maintainers
* Organizations with investments in Node.js 

We also documented what we thought was important to those “constituencies” in [CONSTITUENCY-NEEDS.md](https://github.com/nodejs/next-10/blob/main/CONSTITUENCY-NEEDS.md).

We think we’ve got a good start, but at this point it only reflects the understanding of the small number of Next-10 team members contributing. At this point we need your help to make sure we’ve got it right and/or update until we do.  You can do that by:

* Taking the [survey](https://www.surveymonkey.com/r/86SSY9Q)
* Getting involved in the [Next-10](https://github.com/nodejs/next-10) effort.

Thanks for reading and we hope to get your feedback through the survey or see you get involved in the ongoing work of the Next-10 team. Thanks in advance for your help.

**Comment by mhdawson** [comment]
@nodejs/next-10 I'm forwarding this on to Rachel on the Foundation team to go along with the survey mentioned in https://github.com/nodejs/next-10/issues/48

Please take a look and give me any comments/suggestions so that we can be ready for the survey to go out soon.

**Comment by GeoffreyBooth** [comment]
> So far the team has spent most of its team

Most of its time?

**Comment by mhdawson** [comment]
@GeoffreyBooth thanks, fixed.

**Comment by favasconcelos** [comment]
@mhdawson I guess the link here for the survey is not up-to-date, right? 
Should it be https://www.surveymonkey.com/r/86SSY9Q ?

**Comment by trivikr** [comment]
The link is accurate in the blog post: [Next 10 years of Node.js](https://nodejs.medium.com/next-10-years-of-node-js-understanding-the-needs-of-the-node-js-constituencies-2f95a1df6a6f)

**Comment by mhdawson** [comment]
Added the link in the original post just in case people look there.

**Comment by GeoffreyBooth** [comment]
I took a look at the survey. In general it's pretty good, but I was a bit confused by the part that asked which priorities were important to me. Like, almost all are important, but not all equally so. I feel like this survey would generate more informative data if each priority was listed with a scale next to it, 1 to 5 not at all important to very important, something like that.

**Comment by Songkeys** [comment]
1. The link below is not correct.

> Getting involved in the [Next-10](https://github.com/nodejs/next-10.) effort.

2. I agree with @GeoffreyBooth that we should have a scale for these options. Actually, I took a look at the question and the options are so many that scares me to make a choice...

3. Has this published already? I saw the medium post above. What is the due date of this survey then?

4. Do you consider translating this into different languages? Not just in the English Internet?

**Comment by mhdawson** [comment]
The blog is already published and the survey is live.  We don't have a date to close the survey yet.  

Thinking about translating surveys and potentially using a scale as suggested above is good feedback that we can think about for future surveys.

**Comment by mhdawson** [comment]
Opened this issue so that we discuss the issue of translations - https://github.com/openjs-foundation/user-feedback/issues/6

**Comment by mhdawson** [comment]
Closing as we have published the blog post.

**Comment by bengl** [comment]
Are the results from this somewhere?

**Comment by sheplu** [comment]
@bengl  the results of the survey are now in a PR https://github.com/nodejs/next-10/issues/48#issuecomment-832917723
